uqcsbot/scripts/yt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from uqcsbot import bot, Command
+import argparse
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+YOUTUBE_API_KEY = 'AIzaSyCtxVMs6So6x2WL5WkDV4rm01hzddCCGH4'
+YOUTUBE_API_SERVICE_NAME = 'youtube'
+YOUTUBE_API_VERSION = 'v3'
+YOUTUBE_VIDEO_URL = 'https://www.youtube.com/watch?v='
+
+
+@bot.on_command('yt')
+def handle_yt(command: Command):
+    '''
+    `!yt <QUERY>` - Returns the top search result based on the query string.
+    
+    handle_yt checks the usage of yt command and appropriately selects logic.
+    '''
+    # Makes sure the query is not empty.
+    if command.has_arg():
+        try:
+            cmd = command.arg.strip()
+            yt_normal(cmd, command.channel)
+        except HttpError as e:
+            # Googleapiclient should handle http errors
+            print('An HTTP error %d occurred:\n%s' % (e.resp.status, e.content))
```

Comment:
this would look much nicer with [format strings](https://docs.python.org/3.4/library/string.html#format-string-syntax), i.e. `print(f'A HTTP error {e.resp.status} occured:\n{e.content}')`. Additionally, this should be using `bot.logger.error('blah')` instead of print